elasticgraph-schema_definition/spec/unit/elastic_graph/schema_definition/runtime_metadata/object_types_by_name/update_targets_spec.rb

```diff
@@ -1271,6 +1304,7 @@ def update_targets_for(
               define_relation_and_sourced_from_fields.call(t)
 
               t.index "widgets" do |index|
+                index.has_had_multiple_sources!
```

Comment:
```suggestion
                index.has_had_multiple_sources! if t.fields_with_sources.any?
```

It's nice to only call `has_had_multiple_sources!` when it's needed. (There are a handful of tests in this file that don't want up defining `sourced_from` fields, and `has_had_multiple_sources!` isn't needed in that case.)

Reply:
updated